alacritty.yml

```diff
@@ -671,6 +671,7 @@
   #- { key: W,      mods: Command,            action: Quit             }
   #- { key: N,      mods: Command,            action: SpawnNewInstance }
   #- { key: F,      mods: Command|Control,    action: ToggleFullscreen }
+  #- { key: F,      mods: Command,            action: Search           }
```

Comment:
I still need to test this on macOS hardware, just to make sure it doesn't cause any problems.

I don't think there's any obvious default binding for reverse search.

Reply:
Just tested this on my OS/X alacritty install - cmd-f seems to enter search mode, tho I can't seem to find anyway OUT of search mode?   up/down seems to cycle thru my shell history, but <enter> doesn't seem to process/execute anything - have to restort to closing the window...

Reply:
Actually - this was tested using a `master` build, not this PR.

Reply:
Escape should leave the vi mode. Command + F is identical to what Firefox uses on macOS, correct? Are you aware of any standard binding for searching in the opposite order? I know ctrl + command + F is already used for fullscreen, is command + shift + F already in use?

Reply:
I just pulled the viless PR/branch which I didn't have locally and it seems to work fine for me.

I'd added the binding to my own alacritty.yaml file.  There's no current binding for command+shift+F - I see above theres a reference to `ReverseSearch` which doesn't seem to exist?

![alacritty-viless-search](https://user-images.githubusercontent.com/19093/87491531-85c99580-c69c-11ea-8877-45591fcec353.gif)

Reply:
(Removed my custom binding and the default comes along fine.)

Reply:
I haven't added a binding to command+shift+f, because I'm not sure if it is used by any other application. I don't want to add something that is no-standard.

Reply:
I get you!  iTerm2 and the standard OS/X terminal use ctrl-f for find, ctrl-g for find-next and ctrl-shift-g for find previous - which seems to be mapped to enter/shift-enter in this patch?  I think I kinda like enter/shift-enter better - much faster to use anyway.

Reply:
Yes, Enter/Shift+Enter is taken from Firefox's bindings. I think that makes sense to follow.

Reply:
I just want to say - damnit, I've used this new search so much today I'm wondering how I survived without it. Look forward to this hitting the mainline.